src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -130,7 +130,8 @@ public static partial class SymbolFinder
                     {
                         foreach (var interfaceType in GetAllInterfaces(type))
                         {
-                            if (interfaceType.Symbol.MemberNames.Contains(symbol.Name))
+                            if (interfaceType.Symbol.MemberNames.Contains(symbol.Name) ||
+                                symbol.IsPropertyAccessor())
```

Comment:
>IsPropertyAccessor [](start = 39, length = 18)

I had trouble following this change. Could you help me out?

Reply:
yup.  this needs a comment.  We're inside FindImplementedInterfaceMembersAsync, which is basically saying "given a member, try to find out if it was the implementation of some interface member".  

The existing implementatoin works for all member types (up till now), but it failed on property accessors.  That's because we had this little check ```if (interfaceType.Symbol.MemberNames.Contains(symbol.Name) ```, and the MemberNames collection for a named type doesn't contain the synthesized accessor names.

I think that latter behavior is intentional, so the fix here is to jsut add the ```|| symbol.IsPropertyAccessor()``` check to make sure we can find the implemented member.

Reply:
I've clarified the code here now.